### PR TITLE
Remove connection cell input placeholders

### PR DESCRIPTION
**Pull request by aleDsz**
Also, bumping `req_athena` to 0.1.0

### CODE REVIEW

**Review by josevalim — APPROVED** [review]

**Comment by aleDsz** [comment]
> With this, we probably need to revisit the borer right?
> 
> https://github.com/livebook-dev/kino_db/blob/be0c1f613b63fa63786ff5dc0b2c6a038628a5f4/lib/assets/connection_cell/main.css#L79-L81

Yes, I got the red/600 and added some opacity because it was "too red". What if we change with the same color as `kino_maplibre`?

https://github.com/livebook-dev/kino_maplibre/blob/a3b8e46b52f8dead34bf393abbd76afa051b04a4/lib/assets/map_cell/main.css#L74-L76

**Review by jonatanklosko — APPROVED** [review]

File: lib/assets/connection_cell/main.js [hunk 1]
```diff
@@ -108,7 +104,6 @@ export function init(ctx, info) {
         :value="modelValue"
         @input="$emit('update:data', $event.target.value)"
         v-bind="$attrs"
-        :required="required"
```
Comment (jonatanklosko) [inline lib/assets/connection_cell/main.js]: We still need this on the input though?

Comment (aleDsz) [reply lib/assets/connection_cell/main.js]: not anymore, but I removed it before your review I guess 😝

Comment (aleDsz) [reply lib/assets/connection_cell/main.js]: But it will be added to div and input elements

**Pull request merged**